server.js

```diff
@@ -8,9 +8,10 @@
 
 var port = process.env.PORT || 3000;
 
-//require express in our app
+// require express in our app (& bodyParser & Mongoose)
 var express = require('express'),
   bodyParser = require('body-parser');
```

Comment:
You'll want to end this line with a comma instead of a semi-colon since you're not specifying 'var' on the next line for mongoose.